Opc.Ua.CloudLib.Client/UACloudLibClient.cs

```diff
@@ -259,7 +259,6 @@ public async Task<List<Organisation>> GetOrganisationsAsync(int limit = 10, int
                 .AddField(f => f.LogoUrl);
 
             organisationQuery.AddArgument("limit", limit);
-            organisationQuery.AddArgument("offset", offset);
```

Comment:
when we remove the offset, we will lose the paging feature

Reply:
Correct, but for companies/categories we don't need it in my opinion?

Reply:
I'd be ok with keeping these in for uniformity's sake. Not a showstopper from my perspective though.